css/index.css

```diff
@@ -0,0 +1,150 @@
+@import url("../components/nav/index.css");
+
+@import url("../components/button/index.css");
+
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+	/* font-family: "Lato", sans-serif; */
+	font-family: "Montserrat", sans-serif;
+}
+
+body {
+	margin: 0;
+	padding: 0;
```

Comment:
It's better to add reset styling properties in the universal selector.

Reply:
I don't want to set margin and padding zero for all the elements. A personal preference 😅

Reply:
It's okay then.